doc/reference.md

```diff
@@ -787,6 +787,22 @@ class BangExample : StringSpec({
 })
 ```
 
+### SkipTestException
+Sometimes you want to interrupt a test in runtime, as perhaps you don't know at compile-time if the test should be executed. For this, KotlinTest provides a way to interrupt it by throwing an exception: The `SkipTestException`.
+
+```kotlin
+class SkipTestExceptionExample : StringSpec({
+   
+  "Test should be skipped" {
+    if(noBananasEaten()) {
+      throw SkipTestException("This test was skipped due to the amount of bananas I ate today.")
```

Comment:
It's a bit of an odd example.
I'd say something like if (isLocal()) throw SkipTestException("Cannot run this test when in local mode")

Reply:
I randomly came up with a random example.

Your is better. I'll make the change

Reply:
done.